include/IECoreScene/MeshAlgo.h

```diff
@@ -114,15 +119,23 @@ class IECORESCENE_API MeshSplitter
 
 	// Return one of the result meshes.  Because this is an const method which does not alter any shared
 	// data, it is safe to call in parallel on multiple threads
-	MeshPrimitivePtr splitMesh( int id, const IECore::Canceller *canceller = nullptr ) const;
+	MeshPrimitivePtr mesh( int segmentId, const IECore::Canceller *canceller = nullptr ) const;
 
+	// Return the value of the given segment primitive variable corresponding to one of the outputs
 	template< typename T>
 	typename std::vector<T>::const_reference value( int segmentId ) const;
 
 	// Return a bound containing all values of the primitive variable P for one of the result meshes,
-	// or an empty box if there is no P. Yields an identical result to splitMesh( id )->bound(). Faster than
-	// calling splitMesh(), but slower than calling bound() if you already have the split mesh.
-	Imath::Box3f splitBound( int id, const IECore::Canceller *canceller = nullptr ) const;
+	// or an empty box if there is no P. Yields an identical result to mesh( id )->bound(). Faster than
+	// calling mesh(), but slower than calling bound() if you already have the split mesh.
+	Imath::Box3f bound( int segmentId, const IECore::Canceller *canceller = nullptr ) const;
+
+	// TODO - this shouldn't be necessary in general, the caller always has the primitive variable they
```

Comment:
Is this really a todo? I'd just change the comment I think :

```// Used by the python binding, but otherwise shouldn't be necessary```

Alternatively, we could declare the function in the binding to be a friend?

Reply:
I've gone with the updated comment.  I originally tried to make the binding function a friend, but I couldn't figure out how to declare it as a friend without giving the existence of the binding function public visibility.